scripts/JChem.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env jython
+
+import os
+import sys
+
+# Add all JChem jars to path
+JCHEM_LIB = '/Applications/ChemAxon/JChemSuite/lib'
+for f in os.listdir(JCHEM_LIB):
+    sys.path.append(os.path.join(JCHEM_LIB, f))
+
+from chemaxon.formats import MolImporter, MolExporter
+from chemaxon.marvin.io import MolExportException
+
+import common
+
+
+# SMILES import/export options: https://docs.chemaxon.com/display/docs/SMILES+and+SMARTS+import+and+export+options
+# Aromatization and hydrogen export options: https://docs.chemaxon.com/display/docs/Basic+export+options
+
+
+class MyAromaticSmilesWriter(common.AromaticSmilesWriter):
+    def getoutput(self, smi):
+        mol = MolImporter.importMol(smi, 'smiles')
+        try:
+            # a - General (Daylight) aromatization. (Alternatives: a_bas, a_loose, a_ambig, etc.)
+            # u - Write "unique" smiles (include chirality into graph invariants, slower)
+            msmi = MolExporter.exportToFormat(mol, 'smiles:a')
+        except MolExportException as e:
+            # return ' '.join(str(e).split('\n'))  # Write error message to a single line
+            return ''
+        return msmi
+
+
+class MyHydrogenCounter(common.HydrogenCounter):
+    def getoutput(self, smi):
+        mol = MolImporter.importMol(smi, 'smiles')
+        success = mol.dearomatize()
+        if not success:
+            return (None, 'Kekulization_failure')
```

Comment:
Does this do what you want? Was added in https://github.com/nextmovesoftware/smilesreading/pull/1/commits/fd8857ff2e0c2e1a4b3da62e6b3c2fd75a47285d